### PR TITLE
Vault logs separate volume

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/4018

Thoughts?

### CODE REVIEW

File: modules/aws/vault/vault.tf [hunk 1]
```diff
@@ -32,7 +42,7 @@ resource "aws_instance" "vault" {
     volume_size = "${var.volume_size_root}"
   }
 
-  user_data = "${var.user_data}"
+  user_data = "${data.ignition_config.s3.rendered}"
```
[the remark this answers is not in the document]
Comment (corest) [reply modules/aws/vault/vault.tf]: Yes, with 100 bytes :D

File: modules/aws/vault/variables.tf [hunk 2]
```diff
@@ -53,6 +57,11 @@ variable "volume_size_etcd" {
   default = 10
 }
 
+variable "volume_size_logs" {
+  type    = "string"
+  default = 5
```
Comment (kopiczko) [inline modules/aws/vault/variables.tf]: Is it in GB?

What will happen when this volume becomes full?

Comment (corest) [reply modules/aws/vault/variables.tf]: Yes, it is in gigabytes. If it is full - result will be the same as previously - vault will fail.

Comment (kopiczko) [reply modules/aws/vault/variables.tf]: So why separate partition? So we can SSH?

Do you want to make it bigger afterwards?

Comment (corest) [reply modules/aws/vault/variables.tf]: Yes, we can adjust it later. But keeping logs separately is good idea in general.

**Pull request merged**